test/renate/data/test_data_module.py

```diff
@@ -131,7 +131,7 @@ def test_tiny_imagenet_data_module(tmpdir):
     assert isinstance(val_data, Dataset)
     assert len(test_data) == num_te
     assert isinstance(test_data, Dataset)
-    assert train_data[0][0].size() == test_data[0][0].size() == (3, 64, 64)
+    assert train_data[0][0].size == test_data[0][0].size == (64, 64)
```

Comment:
can we drop the second `[0]` and check for `(3, 64, 64)`?

Reply:
No. `train_data[0]` is the first dataset element, and `train_data[0][0]` is a PIL Image of size 64x64.